models/users.js

```diff
@@ -130,11 +142,46 @@ const updateAvatar = async (req, res) => {
         console.log(err.message);
     }
 };
+const searchForUser = async (token) => {
+    try {
+        const { verificationToken } = token;
+        const user = await User.findOne({ verificationToken });
+        if (user) {
+            await User.findOneAndUpdate(
+                { _id: user._id },
+                {
+                    $set: {
+                        verificationToken: null,
+                        verify: true,
+                    },
+                }
+            );
+        }
+        return user;
+    } catch (err) {
+        console.log(err.message);
```

Comment:
```suggestion
        console.log(err);
```
A nie lepiej dowiedzieć się jaka jest cała wiadomość? Czasami message niewiele powie i, i tak trzeba będzie dowiedzieć się więcej